test/fixtures/applications/service-internal-origin/frontend-service/service.yml

```diff
@@ -0,0 +1,8 @@
+type: public
+description:
+author:
+
+development:
+  port: 3000
```

Comment:
Can we use port 8080 for both services. That way the backend service doesn't have the same host port and container port and we can validate things a bit better